<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replication Manager</string>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="tlw_servers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>546</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <property name="itemsExpandable">
     <bool>false</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Group:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Update period:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Last update:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_srv_group">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>401</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_srv_name">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>401</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_srv_addres">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>401</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lastupdate">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>180</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>401</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_period">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>401</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh_sel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>155</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh now</string>
    </property>
   </widget>
   <widget class="QTreeView" name="tlw_replications">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>217</y>
      <width>530</width>
      <height>301</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <property name="itemsExpandable">
     <bool>false</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="tlw_replications_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>531</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addtask">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>530</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rmtask">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>530</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_starttask">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start_con">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start as continuous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stoptask">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>530</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuServers">
    <property name="title">
     <string>Servers</string>
    </property>
    <addaction name="actionAdd_Server"/>
    <addaction name="actionRemove_Server"/>
    <addaction name="actionEdit_Server"/>
    <addaction name="actionDB_Manager"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionWorkers"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuServers"/>
   <addaction name="menuWindows"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdd_Server">
   <property name="text">
    <string>Add Server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRemove_Server">
   <property name="text">
    <string>Remove Server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEdit_Server">
   <property name="text">
    <string>Edit Server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDB_Manager">
   <property name="text">
    <string>DB Manager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionWorkers">
   <property name="text">
    <string>Workers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
